test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -28,73 +28,6 @@ public class MetricApiTest
         [Fact]
         public void SimpleTest()
         {
```

Comment:
removed as this was not doing any Tests. Will be adding unit tests for all features in subsequent PRs.